File: Bridge/System/Reflection/AssemblyName.cs
namespace System.Reflection
{
    [Bridge.Convention(Member = Bridge.ConventionMember.Field | Bridge.ConventionMember.Method, Notation = Bridge.Notation.CamelCase)]
    public class AssemblyName
    {
        private readonly string displayName;

        public AssemblyName(string assemblyName)
        {
            this.displayName = assemblyName;
        }

        public override string ToString()
        {
            return this.displayName;
        }
    }
}

File: Bridge/Resources/.generated/System/Reflection/AssemblyName.js
    /*System.Reflection.AssemblyName start.*/
    Bridge.define("System.Reflection.AssemblyName", {
        fields: {
            displayName: null
        },
        ctors: {
            ctor: function (assemblyName) {
                this.$initialize();
                this.displayName = assemblyName;
            }
        },
        methods: {
            /*System.Reflection.AssemblyName.toString start.*/
            toString: function () {
                return this.displayName;
            },
            /*System.Reflection.AssemblyName.toString end.*/


        }
    });
    /*System.Reflection.AssemblyName end.*/
